backend/routes/table.js

```diff
@@ -0,0 +1,116 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+
+import connection from '../database';
+
+const router = express.Router();
+
+router.use(bodyParser.urlencoded({ extended: true }));
+
+/**
+ * @swagger
+ *
+ * /table/free:
+ *   get:
+ *     description: Get a list of of hours where there is a table free for a restaurent
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: restaurantID
+ *         description: Primary Key of Restaurant database table
+ *         in: query
+ *         required: true
+ *         type: string
+ *       - name: numberOfGuests
+ *         description: The number of guests that the table is being booked for
+ *         in: query
+ *         required: true
+ *         type: integer
+ *       - name: date
+ *         description: Date for when the booking is made
+ *         in: query
+ *         required: true
+ *         type: string
+ *     responses:
```

Comment:
Can you add 400 as a response code?

Reply:
And 500 -> "Internal server error" is fine

Reply:
And 404

Reply:
Added